sources/packages/web/src/assets/css/student.scss

```diff
@@ -36,3 +36,13 @@
   height: 100%;
   min-height: 350px;
 }
+
+.student-with-clipboard-img {
+  background-image: url("../../assets/images/person-with-clipboard.svg");
```

Comment:
@extend .img-background?

Reply:
yeah, ok. Moved it to formio-shared.scss close to the others.